src/Installers/Windows/RuntimeSiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(RepositoryRoot)\.deps\dependencies.g.props" />
```

Comment:
Both the site extensions and ANCM should probably have something that doesn't rely on a .deps file.